src/App.js

```diff
@@ -1,19 +1,30 @@
 import React from 'react';
-import { Routes, Route } from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
```

Comment:
- [ ] I think here you are using react-router-dom but you forget to install it npm insstall react-router-dom so that make me unable to see your website I highly recommend adding it to your project to let it work properly 🚀💯🔥


![image](https://user-images.githubusercontent.com/58553711/201932858-393d9b9c-ae05-4a0e-a9c0-983a31d47967.png)

Reply:
@Reem-lab I have worked on this bug, and all seems fine without errors but I still can't see my app on the browser. What exactly am I not doing right?